lib/globals/animations.module.scss

```diff
@@ -1,41 +1,75 @@
 @mixin inputFocus() {
-    -o-animation-name: animateFocus;
-    -moz-animation-name: animateFocus;
-    -webkit-animation-name: animateFocus;
-    animation-name: animateFocus;
-    @keyframes animateFocus {
-        0% {
-            width: 0%;
-        }
-        100% {
-            width: 100%;
-        }
+  -o-animation-name: animateFocus;
```

Comment:
Looks like this was due to prettier. Can you try removing prettier and lint files with eslint and see if it reverts these changes?

Reply:
321bd4ced83925829ae7a889f3d16964b6fa0c12

Reply:
0e0da4dd32e77d0afd2b322cf30853314dd207e8